lib/cdo/github.rb

```diff
@@ -43,6 +43,60 @@ def self.database_changes(pr_number)
     end
   end
 
+  # Creates a new branch with the given name based on the base_branch branch and
+  # merges the given commit into it. If all goes well, pushes the new branch to GitHub.
+  # Note: assumes it is run from an environment with a git worktree called
+  # deploy-management-repo.
+  # @param [String] branch_name The name of the branch to create.
+  # @param [String] commit The the commit that will be merged in. Can either be a sha or
+  #                        a branch name.
+  # @param [String] base_branch The name of the branch that changes will be merged into.
+  def self.create_branch_from_commit(branch_name:, commit:, base_branch:)
+    # check out a new branch based on base_branch
+    prefix_command = 'cd ~/deploy-management-repo'
```

Comment:
thoughts on doing something like this instead of repeatedly changing into the same directory? https://github.com/code-dot-org/code-dot-org/blob/0f5d8651842873d795a7949a746a56c2c05c9323/lib/rake/test.rake#L103

Reply:
Oh that would be way cleaner! Thank you!! I only get a couple hours of dev time each week. If you don't mind, I'd prefer to do that as a follow-up task rather than delaying this whole change by another week or two.